src/main/java/com/github/jreddit/oauth/param/RedditDuration.java

```diff
@@ -0,0 +1,27 @@
+package com.github.jreddit.oauth.param;
+
+/**
+ * Enumerator for the duration of tokens.<br>
```

Comment:
I personally like the highlighting it gives to the JavaDoc instead of having same-level plain text.